tedana/tests/test_tedana.py

```diff
@@ -28,7 +27,7 @@ def compare_nifti(fn, test_dir, res_dir):
     """
     res_fp = (res_dir/fn).as_posix()
     test_fp = (test_dir/fn).as_posix()
-    assert np.allclose(nb.load(res_fp).get_data(), nb.load(test_fp).get_data())
+    assert np.allclose(nib.load(res_fp).get_data(), nib.load(test_fp).get_data())
```

Comment:
Are we consistently using `nib` instead of `nb`, now ? I just want to make sure we don't end up with conflicting imports !

Reply:
I can change it back, but my understanding was that `nib` was the alias used in the nibabel documentation (e.g., [here](http://nipy.org/nibabel/gettingstarted.html#getting-started)). On the other hand, while they may use that alias in at least _some_ of their documentation, I've never seen anything official about it, so we can use `nb` if you prefer.

Reply:
It looks like that's our only import of it for now, and I know I'm the minority opinion so happy to change :) Just wanted to check !